### PR TITLE
Improved logging and tests

### PR DESCRIPTION
**Pull request by danpersa**

### CODE REVIEW

File: src/it/scala/org/zalando/spearheads/innkeeper/routes/PostRoutesStrictSpec.scala [hunk 1]
```diff
@@ -34,72 +34,83 @@ class PostRoutesStrictSpec extends FunSpec with BeforeAndAfter with Matchers {
...
+          response.status should be(StatusCodes.OK)
+          val entity = entityString(response)
+          val route = entity.parseJson.convertTo[RouteOut]
+          route.id should be(1)
+          route.name should be(RouteName(routeName))
+        }
+      }
+    }
 
-        describe("when an invalid name is provided") {
```
Comment (danpersa) [inline src/it/scala/org/zalando/spearheads/innkeeper/routes/PostRoutesStrictSpec.scala]: brackets were mixed in this file :)

File: src/it/scala/org/zalando/spearheads/innkeeper/routes/PostRoutesHostSpec.scala [hunk 2]
```diff
@@ -39,6 +39,17 @@ class PostRoutesHostSpec extends FunSpec with BeforeAndAfter with Matchers {
           route.createdBy should be(UserName("user~1"))
           route.route.matcher.hostMatcher.get should be("www.yahoo.com")
         }
+
+        it("should not create more routes") {
+          val routeName = "route_regex_1"
```
Comment (c00ler) [inline src/it/scala/org/zalando/spearheads/innkeeper/routes/PostRoutesHostSpec.scala]: Do you really need to have it as a separate variable? It is used only in one place. Can be inlined. If needed then named parameters can be used.

File: src/it/scala/org/zalando/spearheads/innkeeper/routes/PostRoutesStrictSpec.scala [hunk 3]
```diff
@@ -34,72 +34,83 @@ class PostRoutesStrictSpec extends FunSpec with BeforeAndAfter with Matchers {
...
 
-          it("should create the new route") {
-            val routeName = "invalid-strict-route-name"
-            val response = postSlashRoutesStrict(routeName, token)
-            response.status should be(StatusCodes.BadRequest)
-          }
+      describe("when an invalid name is provided") {
+        val token = WRITE_STRICT_TOKEN
+
+        it("should create the new route") {
```
Comment (c00ler) [inline src/it/scala/org/zalando/spearheads/innkeeper/routes/PostRoutesStrictSpec.scala]: Description does not match the outcome of the test. BadRequest is returned. I expect that route is not created in this case.

File: src/it/scala/org/zalando/spearheads/innkeeper/routes/PostRoutesStrictSpec.scala [hunk 4]
```diff
@@ -34,72 +34,83 @@ class PostRoutesStrictSpec extends FunSpec with BeforeAndAfter with Matchers {
...
-          }
+        it("should return the 403 Forbidden status") {
+          val response = postSlashRoutesStrict(routeName, token)
+          response.status should be(StatusCodes.Forbidden)
         }
+      }
 
-        describe("when a which doesn't have an associated uid") {
-          val token = "token--employees-route.write_strict"
+      describe("when a which doesn't have an associated uid") {
```
Comment (c00ler) [inline src/it/scala/org/zalando/spearheads/innkeeper/routes/PostRoutesStrictSpec.scala]: I think that it has to be rephrased somehow :)

File: src/main/scala/org/zalando/spearheads/innkeeper/dao/RoutesPostgresRepo.scala [hunk 5]
```diff
@@ -18,12 +19,16 @@ class RoutesPostgresRepo @Inject() (
...
 
+  private val logger = LoggerFactory.getLogger(this.getClass)
+
   private val routesTable = TableQuery[RoutesTable]
 
   private lazy val insertRouteQuery = routesTable returning routesTable.map(_.id) into
     ((routeRow: RouteRow, id) => routeRow.copy(id = Some(id)))
 
   def createSchema: Future[Unit] = {
+    logger.debug(s"create schema")
```
Comment (c00ler) [inline src/main/scala/org/zalando/spearheads/innkeeper/dao/RoutesPostgresRepo.scala]: **s** is not needed in the beginnig

File: src/it/scala/org/zalando/spearheads/innkeeper/routes/PostRoutesHostSpec.scala (hunk 2 above)
Comment (danpersa) [inline src/it/scala/org/zalando/spearheads/innkeeper/routes/PostRoutesHostSpec.scala]: done

File: src/it/scala/org/zalando/spearheads/innkeeper/routes/PostRoutesStrictSpec.scala (hunk 3 above)
Comment (danpersa) [inline src/it/scala/org/zalando/spearheads/innkeeper/routes/PostRoutesStrictSpec.scala]: fixed

File: src/it/scala/org/zalando/spearheads/innkeeper/routes/PostRoutesStrictSpec.scala (hunk 4 above)
Comment (danpersa) [inline src/it/scala/org/zalando/spearheads/innkeeper/routes/PostRoutesStrictSpec.scala]: definitely... fixed :8ball:

File: src/main/scala/org/zalando/spearheads/innkeeper/dao/RoutesPostgresRepo.scala (hunk 5 above)
Comment (danpersa) [inline src/main/scala/org/zalando/spearheads/innkeeper/dao/RoutesPostgresRepo.scala]: changed

**Comment by c00ler** [comment]
ok

**Pull request merged**